Makefile

```diff
@@ -47,5 +47,8 @@ typecheck:
 unittest:
 	recipe/run_test.sh unittest
 
+test_notebooks:
+	recipe/run_test.sh notebooks
```

Comment:
```suggestion
test-notebooks:
	recipe/run_test.sh notebooks
```
Also, please place this block above the `typecheck:` block to preserve the existing lexicographical ordering of the blocks.

Reply:
Fixed this in the latest commit as well.